core/src/main/java/es/urjc/etsii/grafo/solver/algorithms/MultiStartAlgorithm.java

```diff
@@ -0,0 +1,67 @@
+package es.urjc.etsii.grafo.solver.algorithms;
+
+import es.urjc.etsii.grafo.io.Instance;
+import es.urjc.etsii.grafo.solution.Solution;
+
+import java.util.logging.Logger;
+
+/**
+ * Example multistart algorithm, executes a user-defined algorithm until N iterations are reached, return best found.
+ *
+ * @param <S> Solution class
+ * @param <I> Instance class
+ */
+public class MultiStartAlgorithm<S extends Solution<I>, I extends Instance> extends Algorithm<S, I> {
+
+    private static Logger log = Logger.getLogger(SimpleAlgorithm.class.getName());
```

Comment:
FIx: 
`    private static Logger log = Logger.getLogger(SimpleAlgorithm.class.getName());
`
should be: 
`    private static Logger log = Logger.getLogger(MultiStartAlgorithm.class.getName());`